src/Stepper/Stepper.js

```diff
@@ -23,9 +23,9 @@ class Stepper extends Component {
      */
     activeStep: PropTypes.number,
     /**
-     * Should be two or more `<Step />` components
+     * Should be two or more `<Step />` components, but also accepts one.
      */
-    children: PropTypes.arrayOf(PropTypes.node),
+    children: PropTypes.oneOfType([PropTypes.node, PropTypes.arrayOf(PropTypes.node)]),
```

Comment:
Will modify and test locally. Didn't know an array of nodes would also count as a node.
If it works for any amount of children I'll commit it in a minute.